src/content/app/genome-browser/components/track-settings-panel/TrackSettingsPanel.scss

```diff
@@ -2,7 +2,7 @@
 
 .trackSettingsPanel {
   display: grid;
-  grid-template-columns: 140px 250px 1fr;
+  grid-template-columns: max-content max-content 1fr;
```

Comment:
`grid-template-columns: repeat(3, max-content);`, I think. Although that won't work with Andrea's 4-column variation panel design.

Do we even need a grid here? Can we do flex (if we need vertical alignment) or just plain old display: block containing display: inline-block (if we don't)?

Reply:
repeat sounds good. Can't we just use repeat(4, max-content)

Reply:
I think flex for this case will be superior. You aren't creating any grid here.

Reply:
Yes I can replace it with flex.
Although with repeat max-content, we haven't added any gutter space so it is still fine. But yes we are not creating a grid layout here.